application/classes/Ushahidi/Formatter/Tos.php

```diff
@@ -0,0 +1,17 @@
+<?php defined('SYSPATH') OR die('No direct access allowed.');
+
+/**
+ * Ushahidi API Formatter for CSV
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Application
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+use Ushahidi\Core\Traits\FormatterAuthorizerMetadata;
+
+class Ushahidi_Formatter_Tos extends Ushahidi_Formatter_API
+{
+    use FormatterAuthorizerMetadata;
```

Comment:
This should have formatters for the dates - have a look at `Ushahidi_Formatter_API` for what we use for created/updated dates